fax/competitive/cga.py

```diff
@@ -1,5 +1,6 @@
 import collections
 from functools import partial
+from typing import Tuple
```

Comment:
Since this is not related to extragradient, perhaps better to slice into separate PR